main.go

```diff
@@ -527,6 +528,17 @@ func mainCore() error {
 	default:
 	}
 
+	quitCleanUpTicker := make(chan int)
+	defer func() {
+		// safely exit the clean up ticker
+		quitCleanUpTicker <- 1
+	}()
+
+	// Instantiates the ticketpool data cache and set it with the current
+	// block height. Also Lazily cleans up the invalid data in the tickepool
+	// cache when a new block is added.
+	go explore.CleanUpTicketPoolData(quitCleanUpTicker)
```

Comment:
You could use the already existing `quit` channel.

Reply:
ok. Noted